### PR TITLE
update kpanda-create-services

### PR DESCRIPTION
**Pull request by ModetaNiu**
#4817
Require review @lemontrees123 @qiupingdai ~Thanks

### CODE REVIEW

File: docs/zh/docs/kpanda/user-guide/network/create-services.md [hunk 1]
```diff
@@ -26,9 +26,9 @@
 
         也可以通过 YAML 创建 一个服务。
 
-3. 打开 __创建服务__ 页面，选择一种访问类型，参考以下三个参数表进行配置。
+3. 打开 __创建服务__ 页面，选择一种访问类型，参考以下四个参数表进行配置。
```
Comment (windsonsea) [inline docs/zh/docs/kpanda/user-guide/network/create-services.md:29]:
```suggestion
3. 打开 __创建服务__ 页面，选择一种访问类型，参考以下几个参数表进行配置。
```
可以模糊处理，万一 UI 又增删了呢

File: docs/zh/docs/kpanda/user-guide/network/create-services.md [hunk 2]
```diff
@@ -66,17 +66,28 @@
...
 | 标签选择器    | 【类型】必填<br />【含义】添加标签，Service 根据标签选择 Pod，填写后点击“添加”。也可以引用已有工作负载的标签，点击 __引用负载标签__ ，在弹出的窗口中选择负载，系统会默认将所选的负载标签作为选择器。 |          |      |
 | 负载均衡类型  | 【类型】必填<br />【含义】使用的负载均衡类型，当前支持 MetalLB 和其他。 |          |      |
 | MetalLB IP 池 | 【类型】必填<br />【含义】选择的 负载均衡类型为 MetalLB 时，LoadBalancer Service默认会从这个池中分配 IP 地址, 并且通过 APR 宣告这个池中的所有 IP 地址，详情请参考：[安装 MetalLB](../../../network/modules/metallb/install.md) |          |      |
 | 负载均衡地址  | 【类型】必填<br />【含义】<br />1.如使用的是公有云 CloudProvider，此处填写的为云厂商提供的负载均衡地址；<br />2.如果上述负载均衡类型选择为 MetalLB ，默认从上述 IP 池中获取 IP ，如果不填则自动获取。 |          |      |
 | 端口配置      | 【类型】必填<br />【含义】为服务添加协议端口，需要先选择端口协议类型，目前支持 TCP、UDP 两种传输协议。<br />**端口名称**：输入自定义的端口的名称。<br />**服务端口（port）**：Pod 对外提供服务的访问端口。默认情况下，为了方便起见，服务端口被设置为与容器端口字段相同的值。<br />**容器端口（targetport）**：工作负载实际监听的容器端口。<br />**节点端口（nodeport）**：节点的端口，接收来自 ClusterIP 传输的流量。用来做外部流量访问的入口。 |          |      |
 | 注解          | 【类型】选填<br />【含义】为服务添加注解<br />               |          |      |
 
+### 创建 ExternalName 服务
+
+点选 __外部服务（ExternalName）__ ，这是指通过将服务映射到外部域名来暴露服务。选择此项的服务不会创建典型的集群内部 IP 或端口，而是通过 DNS 名称解析将请求重定向到外部的服务地址。参考下表配置参数。
```
Comment (windsonsea) [inline docs/zh/docs/kpanda/user-guide/network/create-services.md:82]:
```suggestion
点选 __外部服务（ExternalName）__ ，这是指通过将服务映射到外部域名来暴露服务。选择此项的服务不会创建典型的 ClusterIP 或 NodePort，而是通过 DNS 名称解析将请求重定向到外部的服务地址。参考下表配置参数。
```

**Comment by lemontrees123** [comment]
服务列表图片打不开，建议上传后引用

**Comment by windsonsea** [comment]
See preview: 
https://deploy-preview-4842--daocloud-docs.netlify.app/kpanda/user-guide/network/create-services

**Review by windsonsea — APPROVED** [review]

**Pull request merged**